templates/frontend/pages/contact.tpl

```diff
@@ -0,0 +1,128 @@
+{**
+ * templates/frontend/pages/contact.tpl
```

Comment:
Quite a bit of this could also be shared in `lib-pkp`. But I think we have bigger plans for the whole About area to turn it into a set of defaults managed by CMS-like tools, so maybe better not to finesse it too much in its current state.